oak_dice/BUILD

```diff
@@ -23,6 +23,9 @@ package(
 rust_library(
     name = "oak_dice",
     srcs = glob(["src/**"]),
+    crate_features = [
+        "no_std",
```

Comment:
Interesting, so is `no_std` treated like a crate feature by these rules? I wonder what happens if the crate also defines an actual feature with the same name.

Reply:
Yes with Bazel you define the build options where you declare the package, not where you import it. That's one of the things that might mean defining different rules for different option. We already have that here internally: third_party/oak/oak_dice/BUILD , line 16.